plugins/orca_security/unit_test/parameters/get_asset_by_id.json.resp

```diff
@@ -0,0 +1,192 @@
+{
+  "parameters": [
+    [
+      "found",
+      "test-asset-123",
+      {
+        "asset": {
+          "group_val": "group",
+          "asset_type_string": "VM",
+          "configuration": {},
+          "group_type_string": "VM",
+          "group_type": "asg",
+          "cluster_type": "asg",
+          "type": "vm",
+          "group_unique_id": "group-12345",
+          "tags_value_list": [
+            "ECSAutoScalingGroup"
+          ],
+          "vm_asset_unique_ids": [
+            "vm_12345"
+          ],
+          "cloud_account_id": "44d88612-fea8-a8f3-6de8-2e1278abb02f",
```

Comment:
I believe all the values are faked in `.json.resp` files 😄

Reply:
Yes, all sensitive data is not real :)